contrib/clojure-package/src/org/apache/clojure_mxnet/image.clj

```diff
@@ -60,14 +63,38 @@
   ([input-stream]
    (decode-image input-stream {})))
 
+(defn decode
+  "Decodes an image from an input stream with OpenCV.
+    `input-stream`: `InputStream` - Contains the binary encoded image
+    `color-flag`: 0 or 1 - Convert decoded image to grayscale (0) or color (1)
```

Comment:
one thing i'm wondering is whether `color-flag` should be specified as `:grayscale` or `:color` and we handle the conversion to 0, 1 in this function?

Reply:
Very good point! Let me fix this then.

Reply:
I had to use another name for the keyword `:color` instead of `:color-flag`. I am open to better name for that. The problem is that it is not possible to have two different specs with the same name.